doc/classes/RandomNumberGenerator.xml

```diff
@@ -6,6 +6,13 @@
 	<description>
 		RandomNumberGenerator is a class for generating pseudo-random numbers. It currently uses [url=http://www.pcg-random.org/]PCG32[/url].
 		[b]Note:[/b] The underlying algorithm is an implementation detail. As a result, it should not be depended upon for reproducible random streams across Godot versions.
+		To generate a random float number (within a given range) based on a time-dependant seed:
+		[codeblock]
+		var rng = RandomNumberGenerator.new()
+		func _ready():
+		    rng.randomize()
+		    var my_random_number = rng.randf(-10.0, 10.0)
```

Comment:
I missed it earlier, but this is also wrong. `randf()` returns a float from 0 to 1.0, it's `randf_range()` which lets you specify min and max.

Reply:
oops...I was using Atom when editing this